oteapi_dlite/strategies/parse_image.py

```diff
@@ -0,0 +1,195 @@
+    - `("mediaType", "image/png")`
+    - `("mediaType", "image/tiff")`
+
+    """
+
+    META_PREFIX = "http://onto-ns.com/meta/1.0/generated_from_"
+    parse_config: "ResourceConfig"
+
+    def initialize(
+        self, session: "Optional[Dict[str, Any]]" = None
+    ) -> "Dict[str, Any]":
+        """Initialize."""
+        return {}
+
+    def get(self, session: "Optional[Dict[str, Any]]" = None) -> "Dict[str, Any]":
+        """Execute the strategy.
+
+        This method will be called through the strategy-specific endpoint
+        of the OTE-API Services.
+
+        Parameters:
+            session: A session-specific dictionary context.
+
+        Returns:
+            DLite instance.
+
+        """
+        from oteapi.datacache.datacache import DataCache
+        from oteapi.strategies.parse.image import ImageDataParseStrategy
+
+        image_config = DLiteImageConfig(**self.parse_config.configuration)
+        if image_config.metadata:
+            raise NotImplementedError(
+                "User-defined metadata for images not implemented"
+            )
+        if "key" not in session:
+            raise RuntimeError("Image parser needs an image to parse")
+        key = session["key"]
+        dc = DataCache()
+
+        if image_config.crop:
+            # Crop the image before creating a DLite datamodel from it
+            # NOTE: Change this when ImageDataParseStrategy.get()
+            # uses datacache key as input
+            suffix = "." + self.parse_config.mediaType.rpartition("/")[2]
+            with dc.getfile(key, suffix=suffix) as tmp_file:
+                tmp_config = self.parse_config.copy()
+                tmp_config.configuration["filename"] = tmp_file.name
+                tmp_config.configuration["localpath"] = tmp_file.parent
+                parsed = ImageDataParseStrategy(tmp_config).get().parsedOutput
+            cropped_file = Path(parsed["cropped_filename"])
+            key = dc.add(cropped_file.read_bytes())
+            cropped_file.unlink()
+
+        with dc.getfile(key) as source:
+            temp = Image.open(source)
+            image = temp.copy()
+            temp.close()
+
+        data = np.asarray(image)
+        if np.ndim(data) == 2:
+            data.shape = (data.shape[0], data.shape[1], 1)
+        meta = __class__.create_meta(
+            image,
+            self.parse_config.mediaType,
+            data.dtype.name,
+        )
+        inst = meta(
+            dims=[image.height, image.width, len(image.getbands())],
+            id=image_config.id,
+        )
+        inst["data"] = data
+        if image.format:
+            inst["format"] = image.format
+        # if image.info:
+        #     inst["info"] = str(image.info)
+        # if "frames" in inst:
+        #     inst["frames"] = getattr(image, "n_frames")
+        #     inst["animated"] = getattr(image, "is_animated", False)
+
+        inst.incref()
+        return {"uuid": inst.uuid}
```

Comment:
For dlite-strategies want to update a collection - lets take that in a separate issue...
```suggestion
        return UpdateSession(uuid=inst.uuid)
```